internal/renderer/service_util.go

```diff
@@ -310,8 +310,9 @@ func createLbService4Gateway(c *RenderContext, gw *gwapiv1a2.Gateway) *corev1.Se
 		svc.Spec.Type = corev1.ServiceTypeLoadBalancer
 	}
 
+	as := gw.GetAnnotations()
```

Comment:
Can we also add the GwConfig.LBSvcAnnotations to this? I mean let's first copy GwConfig.Spec.LBSvcAnnotations` into a new map as and then let's merge `gw.GetAnnotations()` into this map so that the annotations in the latter override the annotations in the former. Just use a simple for loop to do the merge, we need nothing fancy here.

Reply:
I understand what you mean, but don't see the point. The only reason I can see to do this is that this way the mixed proto annotations could be added either to the gateway or to the GatewayConfig's `loadBalancerServiceAnnotations' field. Would you like to achieve this or is there any other reason?

Also in https://github.com/l7mp/stunner-gateway-operator/blob/main/internal/renderer/service_util.go#L344-L352 we copy(add) both annotation lists to the LB Service's annotation list. This could be simplified into a single for loop.